Resources/doc/reference/installation.rst

```diff
@@ -23,10 +23,9 @@ Installation
 
 Retrieve the bundle with composer:
 
-.. code-block:: sh
-
-    php composer.phar require sonata-project/media-bundle --no-update
+.. code-block:: bash
 
+    $ php composer.phar require sonata-project/media-bundle --no-update
```

Comment:
In Symfony we already use `composer require ...` instead of `php composer.phar require ...` But I don't know if the latter is the common Sonata practice.